www/layouts/_default/home.html

```diff
@@ -1,19 +1,19 @@
 {{ define "main" }}
-  {{ .Content }}
+{{ .Content }}
 
-  <h2>Articles récents
+<h2>Articles récents
     <a href="articles/index.xml" title="flux RSS">
-      <img src="/rss.svg" alt="flux RSS" style="height: 1em;"></a>
-  </h2>
+        <img src="/rss.svg" alt="flux RSS" style="height: 0.8em;"></a>
+</h2>
 
-  {{ range first 10 (where site.RegularPages "Section" "articles") }}
-    <article>
-      <header>
+{{ range first 10 (where site.RegularPages "Section" "articles") }}
+<article>
+    <header>
         <h3>{{ .LinkTitle }}</h3>
         <time><a href="{{ .RelPermalink }}">{{ time.Format ":date_medium" .Date }}</a></time>
-      </header>
-      {{ .Summary }}
-      <nav class="more"><a href="{{ .RelPermalink }}">Lire la suite…</a></nav>
-    </article>
-  {{ end }}
+    </header>
+    {{ .Summary }}
+    <nav class="more"><a href="{{ .RelPermalink }}">Lire la suite…</a></nav>
+</article>
 {{ end }}
+{{ end }}
```

Comment:
nitpick, let’s keep an empty line at the end of each file, like true gentlemen do :-)